src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -65,79 +57,46 @@ class BankAccountStep extends React.Component {
     constructor(props) {
         super(props);
 
-        this.toggleTerms = this.toggleTerms.bind(this);
         this.addManualAccount = this.addManualAccount.bind(this);
         this.addPlaidAccount = this.addPlaidAccount.bind(this);
-        this.state = {
-            selectedPlaidBankAccount: undefined,
-            hasAcceptedTerms: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTerms', true),
```

Comment:
qq: @luacmartins because we are not reading from the key `reimbursementAccountDraft`, I am guessing when this gets deployed, this data will be of no use (especially for the keys related to `BankAccountStep`).

and there's nothing to be done to retain the values?

Reply:
What I meant is, we're changing how the draft gets stored due to `Form.js` so any existing data in `reimbursementAccountDraft` will be lost?

Reply:
Yes, that will be the case. I think it's fine. We fetch previous step data when opening the page, so the worst that will happen is losing the current step draft.